lib/sfn/callback/stack_policy.rb

```diff
@@ -74,19 +74,26 @@ def template(info)
       # @param p_stack [Miasma::Models::Orchestration::Stack]
       # @return [NilClass]
       def save_stack_policy(p_stack)
+        valid_logical_ids = p_stack.resources.reload.all.map(&:logical_id)
+        stack_policy = @policies.fetch(p_stack.id,
+          @policies.fetch(p_stack.data[:logical_id]),
+          @policies[p_stack.name]
+        ).to_smash
+        stack_policy[:statement].delete_if do |policy_item|
```

Comment:
This seems to cause an error updating stacks where we're using this callback but haven't explicitly set a stack policy.

```
[Sfn]: Applying stack policy to my-test-stack... error!
[ERROR]: Reason - undefined method `delete_if' for nil:NilClass
ERROR: NoMethodError: undefined method `delete_if' for nil:NilClass
```